### PR TITLE
fix: remove capitalization in subheading

### PR DESCRIPTION
**Pull request by zulnabil**
…o read

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const IndexPage: NextPage = () => (
                 Kabar Gembira!
               </Heading>
               <Heading as="h3" fontSize="5xl" fontWeight={600}>
-                Merchandise Resmi ReactJS&nbsp;Indonesia
```
Comment (zulnabil) [inline pages/index.tsx:34]: image

File: pages/learning.tsx [hunk 2]
```diff
@@ -21,8 +21,8 @@ const LearningPage: NextPage = () => (
           <Heading fontFamily="body" fontWeight="600" as="h2" textTransform="capitalize" fontSize={48} textAlign="center">
```
Comment (resir014) [inline pages/learning.tsx:21]:
```suggestion
          <Heading fontFamily="body" fontWeight="600" as="h2" fontSize={48} textAlign="center">
```

Remove this one as well.

**Review by resir014 — CHANGES_REQUESTED** [review]
@zulnabil LGTM! Just one more thing:

File: pages/learning.tsx (hunk 2 above)
Comment (zulnabil) [reply pages/learning.tsx]: done sir

Comment (zulnabil) [reply pages/learning.tsx]: there is a conflict about the correct use of the word "di"
Screen Shot 2020-10-31 at 19 32 39

**Review by resir014 — APPROVED** [review]

**Pull request merged**